src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageDetails.tsx

```diff
@@ -20,10 +20,14 @@ const WorkPackageDetails: React.FC<WorkPackageDetailsProps> = ({ workPackage })
     <PageBlock
       title={'Work Package Details'}
       headerRight={
-        <>
-          {workPackage.stage ? <WorkPackageStageChip stage={workPackage.stage} /> : null}
-          <WbsStatus status={workPackage.status} />
-        </>
+        <Grid container alignItems="center" justifyContent="flex-end">
```

Comment:
I would try and yarn start this to see what it looks like -- it doesnt look great as the container messes with the way everything looks because it claims to take full width.

So, the way I would approach this is as follows: 

I looked through the codebase and some Grid documentation and its generally not great for this because the way our headerRight works doesn't supplement a container Grid component. Instead, I would use something like a div -- a Box. Generally, we want all our components to be Material UI, and that is why we dont use div, span, etc...

In Material UI, a Box component is like a (essentially) more customizable, better <div>. So, I would make _one_ Box component with the stage chip and the status inside of it, and in justify the content of the Box to the right. Let me know if you have any questions.

Reply:
Generally, it's important you yarn start and checking how it looks before a PR. As you can see the "Work Package Details" gets misformatted with the container. 

<img width="1349" alt="Screen Shot 2023-04-03 at 1 24 24 PM" src="https://user-images.githubusercontent.com/87815804/229582687-a4ea373e-4af6-4d5a-b845-e1d67c13ad56.png">